spec/rubocop/cop/rspec/verified_double_reference_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'verified double references' do |verified_double|
+  let(:cop_class) { RuboCop::Cop::RSpec::VerifiedDoubleReference }
+
+  context 'when EnforcedStyle is constant' do
+    let(:cop_config) do
+      { 'EnforcedStyle' => 'constant' }
+    end
+
+    it 'does not flag a violation when using a constant reference' do
+      expect_no_offenses("#{verified_double}(ClassName)")
+    end
+
+    it 'flags a violation when using a string reference' do
+      expect_offense(<<~RUBY, verified_double: verified_double)
+        %{verified_double}('ClassName')
+        _{verified_double} ^^^^^^^^^^^ Prefer `constant` references for verified doubles.
```

Comment:
Why is "constant" in backticks?
Would "Use a literal constant to refer to a class for verified doubles" be an easier to understand offence hint?

Reply:
Removed the backticks, updated the language to be a compromise between the prior language and your suggestion.